fdbserver/masterserver.actor.cpp

```diff
@@ -170,6 +170,10 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 		recoveryTransactionVersion;  // The first version in this epoch
 	double lastCommitTime;
 
+	Version liveCommittedVersion; // The live committed version reported by proxies.
+	bool proxyLocked;
```

Comment:
This is the `locked` field coming from `ReportRawCommittedVersionRequest`, which means if the database is locked. So I suggest rename as `locked` or `databaseLocked` to avoid confusion.